acceptance/ui/features/hosts.feature

```diff
@@ -91,8 +87,7 @@ Feature: Host Management
       And I click "Cancel"
     Then I should see an empty Hosts page
       And I should not see "Success"
-
-  @login-required
+ @clean_hosts
```

Comment:
style nit - please indent one more space to the right so `@` is aligned with `Scenario`, and add a blank line before `@clean_hosts`. We should always have a blank line between scenarios to help provide a little more visual separation